app/src/main/res/values/dimens.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <dimen name="padding_51dp">51</dimen>
-    <dimen name="size_bank_name_30sp">30sp</dimen>
+    <dimen name="size_bank_name_21sp">20sp</dimen>
     <dimen name="size_sub_heading_20sp">20sp</dimen>
```

Comment:
Try to use a common name like
..dimen name="font_size_20">20sp</dimen..  
to avoid duplicate entry